spec/system/multitenancy_spec.rb

```diff
@@ -0,0 +1,160 @@
+require "rails_helper"
+
+describe "Multitenancy" do
+  before do
+    create(:tenant, subdomain: "mars")
+    create(:tenant, subdomain: "venus")
+  end
+
+  scenario "Disabled features", :no_js do
+    Tenant.switch("mars") { Setting["process.debates"] = true }
+    Tenant.switch("venus") { Setting["process.debates"] = nil }
+
+    with_subdomain("mars") do
+      visit debates_path
+
+      expect(page).to have_css "#debates"
+    end
+
+    with_subdomain("venus") do
+      expect { visit debates_path }.to raise_exception(FeatureFlags::FeatureDisabled)
+    end
+  end
+
+  scenario "Content is different for differents tenants" do
+    Tenant.switch("mars") { create(:poll, name: "Human rights for Martians?") }
+
+    with_subdomain("mars") do
+      visit polls_path
+
+      expect(page).to have_content "Human rights for Martians?"
+      expect(page).to have_css "html.tenant-mars"
+      expect(page).not_to have_css "html.tenant-venus"
+    end
+
+    with_subdomain("venus") do
+      visit polls_path
+
+      expect(page).to have_content "There are no open votings"
+      expect(page).to have_css "html.tenant-venus"
+      expect(page).not_to have_css "html.tenant-mars"
+    end
+  end
+
+  scenario "PostgreSQL extensions work for tenats" do
```

Comment:
Hi @javierm, typo found here:
```suggestion
  scenario "PostgreSQL extensions work for tenants" do
```